app/src/main/java/org/simple/clinic/editpatient/PatientEditValidationError.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.editpatient
+
+enum class PatientEditValidationError(val analyticsName: String) {
+  FULL_NAME_EMPTY("Patient Edit:Name is empty"),
```

Comment:
Same question about making these analytics name more consistent with `PatientEntryValidationError`

Reply:
`PatientEntryValidationError` currently uses `Patient Entry` as the prefix and `PatientEditValidationError` uses `Patient Edit`. Seems like it's more consistent right now.

Reply:
Hrm right. I think the problem is that `PatientEntryValidationError` uses a different analytics name than `PatientEntryScreenEvents`.